pkg/resource/podtemplatespec.go

```diff
@@ -231,6 +231,9 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, proxyLister configl
 	mounts = append(mounts, corev1.VolumeMount{Name: vol.Name, MountPath: "/etc/pki/ca-trust/source/anchors"})
 	deps.AddConfigMap(vol.VolumeSource.ConfigMap.LocalObjectReference.Name)
 
+	// Adds the user provided certificates config map as a dependency.
+	deps.AddConfigMap(v1.TrustedCAConfigMapName)
+
```

Comment:
You can drop this - I'm working on a PR that adds this and mounts it to the right location.